src/distributions/bernoulli.rs

```diff
@@ -48,6 +48,10 @@ impl Bernoulli {
     ///
     /// For `p = 1.0`, the resulting distribution will always generate true.
     /// For `p = 0.0`, the resulting distribution will always generate false.
+    /// Due to the limitations of floating point numbers, not all internally
+    /// supported probabilities (multiples of 2<sup>-64</sup>) can be specified
+    /// using this constructor. If you need more precision, use
```

Comment:
I think it would be better to say:

> This method is accurate for any input `p` in the range `[0, 1]` which is a mutliple of `2^-64`. Values outside this range are treated as if they were `0` or `1` (whichever is nearest).

Reply:
Done.